cloudinit/distros/gentoo.py

```diff
@@ -261,5 +267,13 @@ def convert_resolv_conf(settings):
             result += "nameserver %s\n" % ns
     return result
 
+def convert_resolv_conf_search(settings):
+    """Returns a settings string formatted for resolv.conf."""
+    result = ""
+    if isinstance(settings, list):
```

Comment:
This function has a single callsite and we know that the argument `settings` is a list, so the if statement will always be true and we can remove it.

Reply:
No idea here.  :)

Reply:
Ahh, I fumbled my way through this part.  Guessing the `if` can just be removed then? I wasn't sure there. I copied that code from another part of the script to perform the addition of the search domain(s) IIRC.